kubernetes-ts-helm-wordpress/README.md

```diff
@@ -17,17 +17,11 @@ Helm Chart and submit the expanded YAML to the cluster.
 
 ## Running the App
 
-Use the [Helm installation guide](https://docs.helm.sh/using_helm/#installing-helm) to install the
+Use the [Helm installation guide](https://helm.sh/docs/intro/install/) to install the
 `helm` CLI. On macOS this might look something like:
 
 ```sh
-brew install kubernetes-helm
-```
-
-Once `helm` is installed, initialize it with:
-
-```sh
-helm init --client-only
+brew install helm
```

Comment:
Is this right?  I though the formula was still `kubernetes-helm`?

Reply:
Yes, they renamed the formula